tests/page/locator-click.spec.ts

```diff
@@ -33,10 +33,15 @@ it('should work with Node removed', async ({ page, server }) => {
 });
 
 it('should work for TextNodes', async ({ page, server }) => {
-  await page.goto(server.PREFIX + '/input/button.html');
-  const buttonTextNode = await page.evaluateHandle(() => document.querySelector('button').firstChild);
+  await page.setContent(`
+    <div id='outer' onclick="window['result'] = 'Clicked ' + event.target.id;">
+      <div id='inner' style="max-width: 50px">Lorem ipsum dolor sit amet consectetur adipiscing elit proin, integer curabitur imperdiet rhoncus cursus tincidunt bibendum, consequat sed magnis laoreet luctus mollis tellus. Nisl parturient mus accumsan feugiat sem laoreet magnis nisi, aptent per sollicitudin gravida orci ac blandit, viverra eros praesent auctor vivamus semper bibendum. Consequat sed habitasse luctus dictumst gravida platea semper phasellus, nascetur ridiculus purus est varius quisque et scelerisque, id vehicula eleifend montes sollicitudin dis velit. Pellentesque ridiculus per natoque et eleifend taciti nunc, laoreet auctor at condimentum imperdiet ante, conubia mi cubilia scelerisque sociosqu sem.</div>
+      Custom Text.
+    </div>
+  `);
+  const buttonTextNode = await page.locator('#outer:last-child');
```

Comment:
This targets the outer div, not the text node. In fact, this whole test is just wrong - you cannot target a text node with a locator. Let's remove it.